demo/api.md

```diff
@@ -40,9 +40,9 @@ The auro-select element is a wrapper for auro-dropdown and auro-menu to create a
 
 | Name       | Description                          |
 |------------|--------------------------------------|
-|            | Default slot for the menu content.   |
-| `helpText` | Defines the content of the helpText. |
-| `label`    | Defines the content of the label.    |
+|            | Default slot for the menu content.   |
+| [helpText](#helpText) | Defines the content of the helpText. |
+| [label](#label)    | Defines the content of the label.    |
 <!-- AURO-GENERATED-CONTENT:END -->
 
 ## API Examples

```

Comment:
Update names of the examples to have links that can be clicked from the api list above. See this commit for reference: https://github.com/AlaskaAirlines/auro-drawer/pull/5/commits/83257153e7f98d6c222f271277e58a63c1b54b79